apps/src/templates/codeReviewGroups/CodeReviewGroupsStatusToggle.jsx

```diff
@@ -0,0 +1,108 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import i18n from '@cdo/locale';
+import CodeReviewGroupsDataApi from './CodeReviewGroupsDataApi';
+import {setCodeReviewExpiresAt} from '../../redux/sectionDataRedux';
+import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
+import ToggleSwitch from '@cdo/apps/code-studio/components/ToggleSwitch';
+import color from '@cdo/apps/util/color';
+
+function CodeReviewGroupsStatusToggle({
+  codeReviewExpiresAt,
+  sectionId,
+  setCodeReviewExpiration
+}) {
+  const [saveError, setSaveError] = useState(false);
+  const [saveInProgress, setSaveInProgress] = useState(false);
+  const currentTime = Date.now();
+  const isToggledOn = codeReviewExpiresAt
+    ? codeReviewExpiresAt > currentTime
+    : false;
```

Comment:
nit: could simplify to `codeReviewExpiresAt && codeReviewExpiresAt > currentTime`?

Reply:
this is getting passed to the `ToggleSwitch` component, which expects a boolean, so it needs to be explicitly set for the false case.